lib/netconf/local/genesis.json

```diff
@@ -175,8 +175,10 @@
       "params": {
         "max_withdrawal_per_block": 32,
         "max_sweep_per_block": 128,
-        "min_partial_withdrawal_amount": 100000000
-      }
+        "min_partial_withdrawal_amount": 100000000,
+        "epoch_identifier": "minute"
+      },
+      "epoch_number": "0"
```

Comment:
Is this the starting number of epoch?

Reply:
It is the epoch number currently used in `evmstaking` module for epoch staking. This value is used for checking the next epoch is started in `epochs` module by comparing epoch number.